menas/src/main/scala/za/co/absa/enceladus/menas/controllers/DatasetController.scala

```diff
@@ -44,6 +46,12 @@ class DatasetController @Autowired()(datasetService: DatasetService)
 
   import scala.concurrent.ExecutionContext.Implicits.global
 
+  @GetMapping(Array("/latest"))
```

Comment:
"Standard flow" thorws 500
![image](https://user-images.githubusercontent.com/17467768/128808405-87ab4fcd-b0cd-47df-803c-423e0898595f.png)

No param supplied fails. There should be a default if no param is supplied. Params should not make or break the API, they should just help narrow down what you are looking for.
![image](https://user-images.githubusercontent.com/17467768/128808564-93f5e740-6b40-4eaa-8e3c-1472e55a5dbd.png)

Also the name `menas/api/dataset/latest` tells no story. Latest what? All datasets?

Reply:
Latest dataset versions

Reply:
So in case, no parameter is supplied it should give me all the datasets and their latest version?

Reply:
Yes, it was a controller thing